react-native-libraries.json

```
@@ -10215,6 +10215,7 @@
   },
   {
     "githubUrl": "https://github.com/azesmway/react-native-unity",
+    "npmPkg": "https://www.npmjs.com/package/@azesmway/react-native-unity",
```

```suggestion
    "npmPkg": "@azesmway/react-native-unity",
```
Nice catch, missed that one! 👍 One suggestion, [this field](https://github.com/react-native-community/directory?tab=readme-ov-file#npmpkg) should include the package name string.